documentation/design-docs/dotnet-tools.md

```diff
@@ -551,6 +551,53 @@ UNINSTALL
       Uninstalling SOS from ~/.dotnet/sos
       Complete
 
+### dotnet-gcheapdump
+
+SYNOPSIS
+
+    dotnet-gcheapdump [--version]
+                      [-h, --help]
+                      <command> [<args>]
+
+OPTIONS
+
+    --version
+        Display the version of the dotnet-dump utility.
```

Comment:
```suggestion
        Display the version of the dotnet-gcheapdump utility.
```

Reply:
I'll address this in a wider name change to `gcdump` instead of `gcheapdump`